### PR TITLE
Change the method to make the redirect

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Change the method to perform the redirection when clicking on the button to view the agent or view the agent configuration, so that the view is not broken when refreshing it. 
 
### Issues Resolved
- #5454

### Evidence

image

image

### Test

Navigate to an agent or the configuration of an agent using the table buttons, 
- Make sure that  `tab` and `agent` parameter appears in the url. 
- Refresh the page and the correct information should be displayed.
- In the agent view navigate to `Inventory data | Stats | Configuration` and refresh the page should reload the same information.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: public/controllers/agent/components/agents-table.js
Comment (jbiset) [inline public/controllers/agent/components/agents-table.js:1]: Changing the method to improve redirection leaves the **clickAction** propType unused. Should this proptype be removed?

**Review by jbiset — APPROVED** [review]
### Test

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
|  | :green_circle: | :green_circle: | :black_circle: |

**Details**
<details>
<summary>
:green_circle: Make sure that tab and agent parameter appears in the url.
:green_circle: Refresh the page and the correct information should be displayed.
:green_circle: In the agent view navigate to Inventory data | Stats | Configuration and refresh the page should reload the same information.
</summary>

Chrome - :green_circle:
chrome_inventory_data

chrome_stats

chrome_configuration

Firefox - :green_circle:
firefox_inventory_data

firefox_stats

firefox_configuration

Safari - :black_circle:

</details>

TEST:✔️
CR:✔️ (with single comment)

LGTM

File: public/controllers/agent/components/agents-table.js
Comment (yenienserrano) [reply public/controllers/agent/components/agents-table.js:1]: Done, thanks for the notice

**Review by Desvelao — APPROVED** [review]
**review**

code :green_circle: 
test :green_circle: 

### Test

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
|  | :green_circle: | :green_circle: | :black_circle: |

**Details**
<details>
<summary>
:green_circle: Make sure that tab and agent parameter appears in the url.
:green_circle: Refresh the page and the correct information should be displayed.
:green_circle: In the agent view navigate to Inventory data | Stats | Configuration and refresh the page should reload the same information.
</summary>

Chrome - :green_circle:
image
image
image
image
image

Firefox - :green_circle:
image
image
image
image
image

Safari - :black_circle:

</details>

**Review by jbiset — APPROVED** [review]

**Pull request merged**